### DISCUSSION TITLE
Use spectral signature from one image tile for all subsequent tiles

### DISCUSSION THREAD

**Discussion by adm-gis** [opengeos/segment-geospatial#163] [Q&A] [answered]
I have a large satellite image that overwhelms our GPU (10gb). I was able to tile the image and then loop over the tiles with the generate function. However, since I am only interested in segmenting the building footprints, I'd like to use the predict function, using an input layer. Is there a way to use the input from one tile and apply that input to the subsequent tiles?

Thanks!

**Comment by giswqs** [comment, marked as answer]
Good suggestion. This is not supported yet. I will see if it can be implemented. Contributions are welcome
